src/main/java/com/amazon/janusgraph/diskstorage/dynamodb/iterator/MultiRowParallelScanInterpreter.java

```diff
@@ -59,7 +59,7 @@ public MultiRowParallelScanInterpreter(DynamoDBStore store, SliceQuery sliceQuer
 
     /**
      * This class relies heavily on the behavior of segmented scans with respect to which hash keys are scanned by each segment.
-     * Here's a rough ASCII example to help illustrate:
```

Comment:
still needs reverting, in upstream this reads example: https://github.com/awslabs/dynamodb-janusgraph-storage-backend/blob/master/src/main/java/com/amazon/janusgraph/diskstorage/dynamodb/iterator/MultiRowParallelScanInterpreter.java#L62